Content.Server/Cloning/Components/CloningPodComponent.cs

```diff
@@ -68,7 +68,7 @@ public sealed class CloningPodComponent : Component
         /// amount of biomass needed to clone.
         /// </summary>
         [ViewVariables(VVAccess.ReadWrite)]
-        public float BiomassRequirementMultiplier = 1;
+        public float BiomassRequirementMultiplier = 1.25F;
```

Comment:
I don't think this will work since it's used for upgrading. It gets set after `RefreshPartsEvent` based on component tiers.

Reply:
I might be wrong, but I think changing the parts affects the *current* multiplier, meaning it doesn't set it to constant values. It would scale with the parts as is?

Reply:
That being said, if it's not the case, I'm unsure where the constant values are stored...

Reply:
It uses `PhysicsComponent.FixturesMass`, so base with no upgrades is 1 mass unit = 1 biomass. You'd have to add a separate multiplier `BaseCostPerMassUnit` similar to the biomass reclaimer `BaseYielPerMassUnit`.